client/state/posts/likes/reducer.js

```diff
@@ -104,7 +104,7 @@ export const itemReducer = withSchemaValidation(
 
```

Comment:
At line 100 above (GitHub doesn't let me annotate outside the diff 🙁), there is `state.likeCount`. That looks wrong: there's no `likeCount` property in the state. The count is stored into `found`.

Reply:
Good catch! I will follow-up in another PR to fix this, as it's not directly related.